lib/strong_migrations/adapters/abstract_adapter.rb

```diff
@@ -50,12 +50,32 @@ def target_version(target_version)
         target_version ||= StrongMigrations.target_version
         version =
           if target_version && StrongMigrations.developer_env?
-            target_version.to_s
+            if target_version.is_a?(Hash)
+              if ActiveRecord::VERSION::STRING.to_f < 6.0
+                raise "Multiple databases are not supported by this ActiveRecord version"
+              end
+
+              target_version = target_version.stringify_keys
+              target_version.fetch(db_config_name) do
+                raise "StrongMigrations.target_version is not configured for :#{db_config_name}"
```

Comment:
I think it'd be better to raise a `StrongMigrations::Error` instead of a `RuntimeError`